src/transforms/mapperTransforms.ts

```diff
@@ -397,6 +399,14 @@ function transformDictionaryMapper(pipelineValue: PipelineValue) {
   };
 }
 
+function isDictionarySchema(schema: Schema): schema is DictionarySchema {
```

Comment:
nit: I will not write a seperate function for such a small code. But, it is just personal choice.

Reply:
I like these small functions because they help narrowing down the type. The typescript server in the editor knows if the condition is met and gives you the type without you having to do a manual cast